src/test/pybind/test_rados.py

```diff
@@ -739,6 +739,41 @@ def cb(blah):
         eq(comp.get_return_value(), 0)
         eq(list(self.ioctx.list_objects()), [])
 
+    def test_aio_set_xattr(self):
+        lock = threading.Condition()
+        count = [0]
+        def cb(blah):
+            with lock:
+                count[0] += 1
+                lock.notify()
+            return 0
+        comp = self.ioctx.aio_set_xattr("obj", "key", "value", cb)
```

Comment:
think we should use a byte literal for `value`.

Reply:
@ideepika updated please check.